File: javaLoginCore/src/main/java/com/hamdan/forzenbook/java/login/core/data/database/LoginDatabase.java
package com.hamdan.forzenbook.java.login.core.data.database;

import androidx.annotation.NonNull;
import androidx.room.Database;
import androidx.room.RoomDatabase;

@Database(entities = {LoginEntity.class}, version = 1)
public abstract class LoginDatabase extends RoomDatabase {
    public static final String NAME = "login_db";

    @NonNull
    public abstract LoginDao loginDao();
}

File: loginCore/src/main/java/com/hamdan/forzenbook/login/core/data/database/LoginDatabase.kt
package com.hamdan.forzenbook.login.core.data.database

import androidx.room.Database
import androidx.room.RoomDatabase

@Database(entities = [LoginEntity::class], version = 1)
abstract class LoginDatabase : RoomDatabase() {
    abstract fun loginDao(): LoginDao

    companion object {
        const val NAME = "login_db"
    }
}
